### PR TITLE
Update NFD

### PR DESCRIPTION
**Pull request by kpouget**
* ff00f67 - nfd_deploy: rename into nfd_operator_deploy

---

* 2545bc5 - nfd_undeploy: rename into nfd_operator_undeploy

---

* f62f57b - CI: ci_entrypoint_{sro,nfd-operator}: remove unused case '-*)'

---

* 8aebb77 - nfd: minor fixes of labels/docs/indentation

### CODE REVIEW

File: playbooks/nfd_wait_gpu.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-- name: Deploy NFD Operator from OperatorHub
+- name: Wait for a node to appear with a NVIDIA GPU PCI label
```
Comment (omertuc) [inline playbooks/nfd_wait_gpu.yml:2]: nit: `an NVIDIA`

File: playbooks/nfd_operator_undeploy.yml [hunk 2]
```diff
@@ -4,4 +4,4 @@
   connection: local
   gather_facts: true
   roles:
-    - role: nfd_undeploy
+    - role: nfd_operator_undeploy_from_operatorhub
```
Comment (omertuc) [inline playbooks/nfd_operator_undeploy.yml:7]: role is called nfd_operator_undeploy

Comment (kpouget) [reply playbooks/nfd_operator_undeploy.yml:7]: yes, I missed part of the renaming, solve with the next push

**Comment by kpouget** [comment]
cluster deployment [failed](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift-psap_ci-artifacts/194/pull-ci-openshift-psap-ci-artifacts-master-gpu-operator-e2e/1404721022703243264) ...

/test gpu-operator-e2e

**Comment by kpouget** [comment]
@ArangoGutierrez maybe we can merge this one even if the test fails *because of CI intrastructure flakes*, `master` is broken so it can't make it be worst

(need to unset `ci/prow/gpu-operator-e2e` as _mandatory_ to bypass Github limitation)

**Review by ArangoGutierrez — APPROVED** [review]
/lgtm

**Pull request merged**